src/model/test/Space_GTest.cpp

```diff
@@ -2029,25 +2030,21 @@ TEST_F(ModelFixture, Space_intersectSurfaces_degenerate3) {
   //EXPECT_NEAR(interiorRoofArea, 412.9019, 0.01);
   EXPECT_EQ(numRoofSurfaces, 9);
 
-  //m.save("intersect3.osm", true);
   //openstudio::path outpath = resourcesPath() / toPath("model/Space_intersectSurfaces_degenerate3_after_intersect.osm");
   //m.save(outpath, true);
 }
 
-// Digital Alchemy
-#ifdef WIN32
-/// <summary>
-/// Illustrates a fix for surface intersection getting stuck in a loop
-/// First of all we need to remove surfaces that overlap within the same space
-/// Second of all we use a different removeSpikes method that shrinks and expands the polygon
-/// </summary>
-/// <param name=""></param>
-/// <param name=""></param>
+// Illustrates a fix for surface intersection getting stuck in a loop
+// First of all we need to remove surfaces that overlap within the same space
+// Second of all we use a different removeSpikes method that shrinks and expands the polygon
 TEST_F(ModelFixture, RemoveSpikesAndOverlaps_TZ46_TZ47) {
   Model model;
   openstudio::path path = resourcesPath() / toPath("/model/RemoveSpikesAndOverlaps_TZ46_TZ47");
 
+#ifdef WIN32
 #  pragma region SPACE 1(TZ46 - 81)
+#endif
+
```

Comment:
This is necessary because GCC doesn't udnerstand `#pragma region` which is a MSVC thing (clang does support it though)

Reply:
Honestly I'd probably just remove that altogether, but whatever